app/home.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1 user-scalable=no">
     <title>MetaMask</title>
+    {{@if(it.isMMI)}}
```

Comment:
we will also need to check if it's not MMI, for the remaining cases. Because if not we will end up with 2 <title> when isMMI is true.